src/components/Homepage/HomepageStatistics.vue

```diff
@@ -0,0 +1,216 @@
+			// this.content?.media[3] is the 1x video
+			const video = this.content?.media[4] ?? [];
+			return video;
+		},
+		totalLoansInDollarsFormatted() {
+			// Defining empty arrays to put the split contentful strings into
+			const part1 = [];
+			const part2 = [];
+			const contentfulTextString = this.statsBlockText[0].copy;
+			const loansInDollarsFormatted = numeral(this.totalLoansInDollars).format('$0.0a').slice(0, -1);
+			// Splitting the string from returned from contentful, where it finds '{value}'
+			const stringSplit = contentfulTextString.split('{value}');
+			// Pushing the two pieces of the split string the respected arrays
+			part1.push(stringSplit[0]);
+			part2.push(stringSplit[1]);
+			// Joining both pieces of the string and the data from graphql
+			const finalString = part1 + loansInDollarsFormatted + part2;
+
+			return finalString;
+		},
+		repaymentRateFormatted() {
+			// Defining empty arrays to put the split contentful strings into
+			const part1 = [];
+			const part2 = [];
+			const contentfulTextString = this.statsBlockText[1].copy;
+			const repaymentRateFormatted = `${numeral(this.repaymentRate).format('0')}%`;
+			// Splitting the string from returned from contentful, where it finds '{value}'
+			const stringSplit = contentfulTextString.split('{value}');
+			// Pushing the two pieces of the split string the respected arrays
+			part1.push(stringSplit[0]);
+			part2.push(stringSplit[1]);
+			// Joining both pieces of the string and the data from graphql
+			const finalString = part1 + repaymentRateFormatted + part2;
+
+			return finalString;
+		},
+		numCountriesAndLendersFormatted() {
+			// Defining empty arrays to put the split contentful strings into
+			const part1 = [];
+			const part2 = [];
+			const part3 = [];
+			const contentfulTextString = this.statsBlockText[2].copy;
+			const numberOfLendersFormatted = numeral(this.numLenders).format('0.0b').slice(0, -1);
+			// Splitting the string from returned from contentful, where it finds '{value}'
+			const stringSplit = contentfulTextString.split('{value}');
+			// Pushing the three pieces of the split string the respected arrays
+			part1.push(stringSplit[0]);
+			part2.push(stringSplit[1]);
+			part3.push(stringSplit[2]);
+			// Joining all the pieces of the string and the data from graphql
+			const finalString = part1 + this.numCountries + part2 + numberOfLendersFormatted + part3;
+
+			return finalString;
+		},
+		formattedTextStrings() {
+			return [
+				this.totalLoansInDollarsFormatted,
+				this.repaymentRateFormatted,
+				this.numCountriesAndLendersFormatted
+			];
+		}
+	}
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.statistics {
+	&::v-deep em {
+		font-style: normal;
+		color: $kiva-green;
+	}
+
+	&__video-wrapper {
+		&--video {
+			width: 100%;
+			border-radius: rem-calc(10);
+		}
+	}
+
+	&__stat-block {
+		margin-top: rem-calc(40);
+
+		&--icon {
+			margin-bottom: rem-calc(20);
+		}
+
+		&--icon-0 {
```

Comment:
I think you should be able to get rid of these different icon sizes if they're all exported at 48x48 from Figma